sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -265,10 +266,22 @@ final class DataFrameWriter private[sql](df: DataFrame) {
    * @param connectionProperties JDBC database connection arguments, a list of arbitrary string
    *                             tag/value. Normally at least a "user" and "password" property
    *                             should be included.
+   * @param columnMapping Maps DataFrame column names to target table column names.
+   *                           This parameter can be omitted if the target table has/will be
```

Comment:
Indent on subsequent lines looks 5 spaces too deep. Unless there's a compiler reason it won't work, you can simply make it default to an empty `Map`.